### PR TITLE
Goliath children from the Broodmother no longer explode

### PR DESCRIPTION
**Pull request by Goblinu**
## About The Pull Request
This is my first PR so if ive done any mistakes corrections would be appriciated.

This removes the goliath childrens ability to explode after death.

## Why It's Good For The Game

Anyone who has fought the goliath broodmother from the pulsating tumour have experienced the frustration of fighting one. Ever since fastmos was introduced has the goliathbroodmother been borderline unbeatable if a player who knows how to use it gets in controll of it.  

The mayor reason as to why the goliath broodmother is a hard opponenet is not due to any of its abilities, but rather the small childrens ability to explode upon death. The explosion is small and causes little to no damage, but flings you across the map as if you got launched from a mass driver. Most of the time this fight is just the goliath brood mother spawning the children, kiting away from the miner constantly knowing that if you kill one of the children you got flinged into a obstacle or a child and gets stunned, surrounded, and then die. The broodmother is strong enough without this ability, and does not need this at all to be a challange to the player. 

## Changelog
:cl:

del: Removes Post-death explosion on goliath children

/:cl:

### CODE REVIEW

**Comment by jupyterkat** [comment]
remove the line that calls explosion() only please

File: code/modules/mob/living/simple_animal/hostile/mining_mobs/elites/goliath_broodmother.dm [hunk 1]
```diff
@@ -189,14 +189,6 @@
 		visible_message("<span class='warning'>[src] digs one of its tentacles under [target]!</span>")
 		new /obj/effect/temp_visual/goliath_tentacle/broodmother(tturf, src)
 
-/mob/living/simple_animal/hostile/asteroid/elite/broodmother_child/death()
-	. = ..()
-	if(mother != null)
-		mother.children_list -= src
```
Comment (utilbird) [inline code/modules/mob/living/simple_animal/hostile/mining_mobs/elites/goliath_broodmother.dm]: This part shouldn't be removed

**Comment by bloons3** [comment]
Isn't this actually caused by Hollywood explosions and not fast atmos?

**Comment by ItsTisme** [comment]
Ngl the hardstun is a gaint meme and makes broodmother 10* worse to fight then any other lavaland elite. Though the children on its own are really weak. Maby release some gas  upon death?

**Comment by Goblinu** [comment]
> 
> 
> Ngl the hardstun is a gaint meme and makes broodmother 10* worse to fight then any other lavaland elite. Though the children on its own are really weak. Maby release some gas upon death?

The broodmother can spawn 2 children every 4 seconds, it justifies them being weak

**Comment by ItsTisme** [comment]
> 
> The broodmother can spawn 2 children every 4 seconds, it justifies them being weak
Maby if brood mom had some better alternatives

There default punch does not even 5 damage and the tenticals are trivial to dodge. Not to mention brood mother is super slow.

They will still have to resort to kiting and cheesing just as much but only diffrence is that the kiting it alot more difficult to do

**Comment by Goblinu** [comment]
> 
> 
> > The broodmother can spawn 2 children every 4 seconds, it justifies them being weak
> > Maby if brood mom had some better alternatives
> 
> There default punch does not even 5 damage and the tenticals are trivial to dodge. Not to mention brood mother is super slow.
> 
> They will still have to resort to kiting and cheesing just as much but only diffrence is that the kiting it alot more difficult to do

The brood mother will now be as hard as any of the other tumour fauna, they arent supposed to be harder than a megafauna to kill

File: code/modules/mob/living/simple_animal/hostile/mining_mobs/elites/goliath_broodmother.dm [hunk 2]
```diff
@@ -188,14 +188,11 @@
 	if(get_dist(src, target) <= 7)//Screen range check, so it can't attack people off-screen
 		visible_message("<span class='warning'>[src] digs one of its tentacles under [target]!</span>")
 		new /obj/effect/temp_visual/goliath_tentacle/broodmother(tturf, src)
-
+
```
Comment (park66665) [inline code/modules/mob/living/simple_animal/hostile/mining_mobs/elites/goliath_broodmother.dm]:
```suggestion

```

**Comment by utilbird** [comment]
This doesn't really balance broodmother, it makes them trivial instead of very difficult, the former is arguably worse.

**Comment by Goblinu** [comment]
> 
> 
> This doesn't really balance broodmother, it makes them trivial instead of very difficult, the former is arguably worse.

What would be a good compromise then?

**Comment by utilbird** [comment]
> What would be a good compromise then?

I don't really know, honestly. Maybe something like an acid or fire gas release?

**Review by utilbird — APPROVED** [review]

**Comment by utilbird** [comment]
This one passed vote so I'll merge it anyways

**Pull request merged**